force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -0,0 +1,185 @@
+import { LightningElement, track, wire, api } from "lwc";
+import { getObjectInfo } from "lightning/uiObjectInfoApi";
+import { format } from "c/util";
+import TOTAL_RECORDS_LABEL from "@salesforce/label/c.Total_Records";
+import ADD_RECORD_LABEL from "@salesforce/label/c.Add_Records";
```

Comment:
label should be c.Add_Record not Records since on screen 3 we will need Add Records plural as well.